### PR TITLE
Adding verifier docker-compose

### PR DESCRIPTION
**Pull request by willmeister**
## Description
Adds verifier setup pointing at publicly-available dockerhub images.

Note: Geth tx ingestion does not work properly, so verifier is not functional.

## Questions
- 
-
-

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: docker-compose.verifier.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      - RUN_GETH_SUBMISSION_QUEUER=1                                                                # Set to anything to run Geth Submission Queuer
+      - RUN_QUEUED_GETH_SUBMITTER=1                                                                 # Set to anything to run Queued Geth Submitter
+      - RUN_FRAUD_DETECTOR=1                                                                        # Set to anything to run Fraud Detector
+      # Sequencer-only Services
+      - RUN_CANONICAL_CHAIN_BATCH_CREATOR=                                                          # Set to anything to run Canonical Chain Batch Creator
+      - RUN_CANONICAL_CHAIN_BATCH_SUBMITTER=                                                        # Set to anything to run Canonical Chain Batch Submitter & Finalizer
+      - RUN_STATE_COMMITMENT_CHAIN_BATCH_CREATOR=                                                   # Set to anything to run State Commitment Chain Batch Creator
+      - RUN_STATE_COMMITMENT_CHAIN_BATCH_SUBMITTER=                                                 # Set to anything to run State Commitment Chain Batch Submitter & Finalizer
```
Comment (karlfloersch) [inline docker-compose.verifier.yml:68]: When you leave these unset it doesn't cause any errors or anything right?

**Review by karlfloersch — APPROVED** [review]
LGTM! I just commented a question but this looks great!!!

File: docker-compose.verifier.yml (hunk 1 above)
Comment (willmeister) [reply docker-compose.verifier.yml:68]: Nope -- just left them in for more info.

Comment (karlfloersch) [reply docker-compose.verifier.yml:68]: Cool!!!

**Pull request closed without merging**